client/src/components/routing/Routes.js

```diff
@@ -11,7 +14,10 @@ const Routes = () => {
     <section className='container'>
       <Switch>
         <Route exact path='/' component={Landing} />
-        <PrivateRoute exact path='/admin_dashboard' component={AdminDashboard} />
+        <Route exact path='/admin_dashboard' component={AdminDashboard} />
```

Comment:
What exactly is the difference between Route and PrivateRoute? Shouldn't anything behind both the admin dash and instructor dash be private routes?

Reply:
Good catch! I change the privacy for testing and forgot to put it back.